### ISSUE TITLE
Feature request: OBS support

### ISSUE THREAD

**Issue by fluffy-critter** [bdkjones/MuzzleApp#11]
As embarrassing as it is to have a notification come up while sharing a screen with a colleague, imagine how much worse it is if you're streaming to the Internet at large.

**Comment by bdkjones** [comment]
I don't know what oBS is.

**Comment by sariyamelody** [comment]
OBS is https://obsproject.com/ -- an open-source live-streaming software suite.

**Comment by fluffy-critter** [comment]
Yes, sorry, should have been more clear about that. It's very popular software for live streaming on twitch.tv and Picarto and the like.

**Comment by Naismith** [comment]
+1 for OBS support. As it has both the ability to live stream to different platforms as well as record. No different than other screen sharing software.

**Comment by fluffy-critter** [comment]
It’s worth noting that Mojave’s built in “do not disturb” functionality seems to turn on when screen sharing is enabled and OBS now appears to be a screen share. So this might not be necessary anymore?

**Comment by bdkjones** [comment]
Not to my knowledge. Mojave will turn on DND when you hook up a projector or use AirPlay, but that’s not screensharing.

**Comment by fluffy-critter** [comment]
Ah, you're right, the reason DND has been automatic for me lately is because in my streaming setup I have a second monitor set to mirror (for reasons not worth getting into) and that's what's been disabling it.
